### PR TITLE
Fix PJSUA2 documentation

### PR DESCRIPTION
**Pull request by wosrediinanatour**
This commit fixes copy&past errors from PJSUA, uses enums instead of magic numbers, and some minors.

### CODE REVIEW

File: pjsip/include/pjsua2/call.hpp [hunk 1]
```diff
@@ -1051,8 +1051,8 @@ struct OnCallRxReinviteParam
     
     /**
      * Status code to be returned for answering the offer. On input,
-     * it contains status code 200. Currently, valid values are only
-     * 200 and 488.
+     * it contains status code PJSIP_SC_OK. Currently, valid values are only
+     * PJSIP_SC_OK and PJSIP_SC_NOT_ACCEPTABLE_HERE.
```
Comment (sauwming) [inline pjsip/include/pjsua2/call.hpp:1055]: Perhaps it's just me, having worked with SIP for so long, the number sticks more to me than the text. So, I'd suggest not to remove the number, i.e.
`PJSIP_SC_OK (200)` and `PJSIP_SC_NOT_ACCEPTABLE_HERE (488)`.

Also for other instances above and below.

File: pjsip/include/pjsua2/call.hpp [hunk 2]
```diff
@@ -2014,27 +2014,25 @@ class Call
      *
      * @param prm       Callback parameter.
      */
-    virtual void onCallReplaced(OnCallReplacedParam &prm)
-    { PJ_UNUSED_ARG(prm); }
+  virtual void onCallReplaced(OnCallReplacedParam &prm) { PJ_UNUSED_ARG(prm); }
```
Comment (sauwming) [inline pjsip/include/pjsua2/call.hpp:2017]: why does this have to be squeezed into one line?

File: pjsip/include/pjsua2/call.hpp [hunk 3]
```diff
@@ -1512,7 +1512,7 @@ class Call
      * which means for incoming call, this function can be called as soon as
      * call is received as long as incoming call contains SDP, and for outgoing
      * call, this function can be called only after SDP is received (normally in
-     * 200/OK response to INVITE). As a general case, application should call
+     * PJSIP_SC_OK/OK response to INVITE). As a general case, application should call
```
Comment (sauwming) [inline pjsip/include/pjsua2/call.hpp:1515]: for this case, 200/OK is already correct. no modification needed.

File: pjsip/include/pjsua2/call.hpp (hunk 2 above)
Comment (wosrediinanatour) [reply pjsip/include/pjsua2/call.hpp]: This was done by accident.

**Comment by wosrediinanatour** [comment]
Updated commit due to your feedback. Please take a look.

File: pjsip/include/pjsua2/call.hpp [hunk 4]
```diff
@@ -1882,8 +1881,8 @@ class Call
 
     /**
      * Notify application when an audio media session is about to be created
-     * (as opposed to on_stream_created() and on_stream_created2() which are
-     * called *after* the session has been created). The application may change
+     * (as opposed to onCallSdpCreated(), which is called *after* the session
```
Comment (sauwming) [inline pjsip/include/pjsua2/call.hpp:1884]: should be `onStreamCreated()`

**Review by sauwming — APPROVED** [review]

**Review by nanangizz — APPROVED** [review]

**Pull request merged**